src/app/jscaip/TableUtils.ts

```diff
@@ -72,6 +72,23 @@ export class TableUtils {
         return total;
     }
 
+    /**
+     * We will consider than in each sub array of this tile, index 0 is the leftmost index
```

Comment:
in each line of this table.

Par contre, si t'as un match en (0, 0), et en (0, 2), tu retourne que le (0, 0). Donc c'est "one of the leftmost", pas juste "leftmost". C'est actually "the upper-left-most", right?

Reply:
comme ça ?